third_party/bigframes_vendored/pandas/core/series.py

```diff
@@ -890,6 +890,90 @@ def groupby(
         used to group large amounts of data and compute operations on these
         groups.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+        You can group by a named index level.
+
+            >>> s = bpd.Series([380, 370., 24., 26.],
+            ...                index=["Falcon", "Falcon", "Parrot", "Parrot"],
+            ...                name="Max Speed")
+            >>> s.index.name="Animal"
+            >>> s
+            Animal
+            Falcon    380.0
+            Falcon    370.0
+            Parrot     24.0
+            Parrot     26.0
+            Name: Max Speed, dtype: Float64
+            >>> s.groupby("Animal").mean()
+            Animal
+            Falcon    375.0
+            Parrot     25.0
+            Name: Max Speed, dtype: Float64
+
+        You can also group by more than one index levels.
+
+            >>> import pandas as pd
+            >>> s = bpd.Series([380, 370., 24., 26.],
+            ...                index=pd.MultiIndex.from_tuples(
+            ...                    [("Falcon", "Clear"),
+            ...                     ("Falcon", "Cloudy"),
+            ...                     ("Parrot", "Clear"),
+            ...                     ("Parrot", "Clear")],
+            ...                    names=["Animal", "Sky"]),
+            ...                name="Max Speed")
+            >>> s
+            Animal    Sky
+            Falcon  Clear     380.0
+                    Cloudy    370.0
+            Parrot  Clear      24.0
+                    Clear      26.0
+            Name: Max Speed, dtype: Float64
```

Comment:
Maybe add a empty line after the output to improve readability.
```suggestion
            Name: Max Speed, dtype: Float64

```

Reply:
Thanks, done!